fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1017,6 +1019,9 @@ public void setCboNetWeight(double cboNetWeight) {
     )
     public boolean ignoreColumnWithComplexType = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_STRONG_CONSISTENCY)
```

Comment:
add `description` field to add both Chinese and English document. See `enableFileCache` for example